packages/fxa-auth-server/lib/senders/renderer/index.ts

```diff
@@ -139,6 +116,29 @@ class Renderer extends Localizer {
     };
   }
 
+  private async getGlobalTemplateValues(
+    context: TemplateContext
+  ): Promise<GlobalTemplateValues> {
+    // make this a switch statement on 'template' if more cases arise?
+    if (context.template === 'lowRecoveryCodes' && context.numberRemaining) {
+      try {
+        // We must use 'require' here, 'import' causes an 'unknown file extension .ts'
+        // error. Might be a config option to make it work?
+        return (
+          await require('../emails/templates/lowRecoveryCodes/includes')
```

Comment:
This doesn't seem super nice at first glance, but, in order to accommodate for something like `lowRecoveryCodes` which requires `numberRemaining` to determine what the subject should be, we need to import a function specific to the template that can work out that logic.

I don't think we'll have to do this very often since it's our only template out of 40something that needs this. I thought about converting all of the `includes.json` files into `includes.ts` for consistency, but it doesn't exactly solve anything. We'd still need an `if` check around which template it is, and to pass in `numberRemaining` if it's `lowRecoveryCodes`.

Also, I went ahead and used `require` here (tried to use `import` but see comment). In #12045, we mention `require`/`import` caches results, and will look at where/how we're using `fetchResource`.

Reply:
IMO, require is alright here, since it needs to be fetched at runtime from inside this function. 

BTW, issues getting the paths to work for storybook / server is tricky and sounds about right. Story book needs to request relative paths from the server... an absolute path like __dirname would be a no go in that context.

Reply:
Ah good, that makes sense and makes me feel better about that path👍